packages/ui/src/components/Tabs/style.less

```diff
@@ -56,4 +56,35 @@
   &-nav-operations {
     display: none !important;
   }
+
+  &-rounded {
+    & > .condo-tabs-nav {
+      .condo-tabs-ink-bar {
+        visibility: hidden;
+      }
+
+      .condo-tabs-tab {
+        padding: @condo-global-spacing-12;
+        border: @condo-global-border-width-default solid @condo-global-color-gray-3;
+        border-radius: 1000px;
```

Comment:
How about just specifying `height / 2` value here (you know height of tab, so you can calculate it), since it's took me a minute, while this magic constant works as expected :)